applications/GeoMechanicsApplication/custom_workflows/dgeosettlement.cpp

```diff
@@ -44,6 +107,14 @@ KratosGeoSettlement::KratosGeoSettlement(std::unique_ptr<InputUtility> pInputUti
         mpGeoApp = Kratos::make_shared<KratosGeoMechanicsApplication>();
         mKernel.ImportApplication(mpGeoApp);
     }
+    if (!mKernel.IsImported("LinearSolversApplication"))
+    {
+        KRATOS_INFO("KratosGeoSettlement") << "Importing LinearSolversApplication" << std::endl;
+        mpLinearSolversApp = Kratos::make_shared<KratosLinearSolversApplication>();
+        mKernel.ImportApplication(mpLinearSolversApp);
+    }
+
+    ParallelUtilities::SetNumThreads(1);
```

Comment:
Can this be decided on by the value in the project parameters file? Jonathan suggested we should use this parameter to test the effect of multi-threading in the future

Reply:
@rfaasse  No problem, good to have it in a future PR